src/renderer/App.tsx

```diff
@@ -102,14 +116,22 @@ export default function App() {
   // game (back button or home button), team users should be navigated to their home screen too. The 'quitGame' message is send to them for
   // this purpose.
   router.subscribe((x) => {
+    // case when you exit game-host screen without finishing game
     if (
       prevLocationRoute &&
       prevLocationRoute === RouteDefinitions.GameHost.route &&
       x.location.pathname !== RouteDefinitions.GameSettings &&
+      x.location.pathname !== RouteDefinitions.GameFinished &&
       !x.matches.some((m) => m.route.path === RouteDefinitions.GameHost.route)
     ) {
       sendMessage('quitGame', currentUser?.id);
     }
+
+    // case when you finish game and exit GameFinished screen
+    if (prevLocationRoute && prevLocationRoute === RouteDefinitions.GameFinished) {
+      sendMessage('quitGame', currentUser?.id);
```

Comment:
same as on line 127, you split it just to understand the cases we have ?

Reply:
yes I split it to be little bit more readable